go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/gdamore/tcell/v2
 
```

Comment:
You've conflated two different things.

go.mod being 1.15 means that we are saying we only support 1.15 and newer.  That's wrong.

We use go1.15 in the workflow because that's what we want to *test* with (because its a pain to test a bunch of ancient stuff)... but we aren't dropping support versions 12 and later.

Please revert this change.

Reply:
Done